src/host/screenInfo.cpp

```diff
@@ -2180,52 +2149,14 @@ void SCREEN_INFORMATION::SetViewport(const Viewport& newViewport,
 // - S_OK
 [[nodiscard]] HRESULT SCREEN_INFORMATION::ClearBuffer()
 {
-    // Rotate the buffer to bring the cursor row to the top of the viewport.
-    const auto cursorPos = _textBuffer->GetCursor().GetPosition();
-    for (auto i = 0; i < cursorPos.y; i++)
-    {
-        _textBuffer->IncrementCircularBuffer();
-    }
-
-    // Erase everything below that point.
-    RETURN_IF_FAILED(SetCursorPosition({ 0, 1 }, false));
-    auto& engine = reinterpret_cast<OutputStateMachineEngine&>(_stateMachine->Engine());
-    engine.Dispatch().EraseInDisplay(DispatchTypes::EraseType::ToEnd);
-
+    _textBuffer->Reset();

```

Comment:
🛑: this was originally preserving the contents of the row the cursor was on, so someone could "Clear buffer" in the terminal, have it blow away everything but the prompt, but leave the prompt on the screen (without the shell doing anything).

now, that's... kinda super messed up in terminal.

(see #1193)